Autopilot/PathManager/Src/trackFinder.cpp

```diff
@@ -0,0 +1,18 @@
+#include <math.h>
+#include "trackFinder.hpp"
+
+double getTrack(double latitudeInitial, double longitudeInitial, double latitudeCurrent, double longitudeCurrent)
```

Comment:
Oh ok gotcha. Will these two points still be as noisy if we take them after sensorFusion? I was thinking this function should be placed at the end of sensorFusion, where GPS locations are already fused with IMU stuff.

Reply:
Ya they will. Sensor fusion, even if really good, isn't enough for what you want to do. in 1/50th of a second, the plane moves very little (maybe about 15 cm). Ask Tony, and he'll likely tell you that his best algorithm will give us accuracies that are of that order of magnitude (maybe 5 or 6 cm and that's being realllllllllllllly optimistic). 5 cm off for a 15 cm travel is dozens of degrees of error in the track.
[10:02 AM]
So your algorithm will report the plane is "thrashing", and the control surfaces will act very vilantly to try to correct errors that don't really exist.
The right way to go about this would be to work with Tony to figure out what the error on his algorithm would be (at least if he can give you an order of magnitude, it may be much worse than what I've estimated here) and from there, figure out how many terms you want to use in your backwards difference approximation.